spec/features/activeadmin-orderable_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.describe 'ActiveAdmin::Orderable', type: :feature do
+  let!(:bottom) { Category.create!(name: 'bottom', position: 0) }
+  let!(:top) { Category.create!(name: 'top', position: 1) }
+  let!(:middle) { Category.create!(name: 'middle', position: 2) }
+
+  before do
+    visit admin_categories_path
+  end
+
+  it 'reorder elements by dragging vertically', js: true do
+    expect(displayed_names).to eq(%w(top middle bottom))
+
+    drag_element(middle, dy: -200)
+
+    expect(displayed_names).to eq(%w(middle top bottom))
+    expect(Category.order(:position).map(&:name)).to eq(%w(middle top bottom))
+  end
+
+  private
+
+  def displayed_names
+    all('.ui-sortable-handle .col-name').map(&:text)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.